packagecore.yaml

```diff
@@ -26,15 +26,28 @@ packages:
         - python3-requests
         - python3-urllib3
         - python3
+# centos no beautifulsoup4
```

Comment:
I used `python-beautifulsoup4` io bytepackager (when it was up) and it worked there. Is this the problem on CentOS that you mentioned earlier?